receiver/telemetrygeneratorreceiver/README.md

```diff
@@ -154,4 +154,15 @@ telemetrygeneratorreceiver:
           resource_attributes:
             host.name: 2ed77de7e4c1
             os.type: linux   
+```       
+### Windows Events Generator
+
+The Windows Events Generator replays a sample of recorded Windows Event Log data. It has no additional configuration, and will ignore `resource_attributes` and `attributes` fields.
```

Comment:
Do we have this restriction on the otlp replay generator? If so we should call it out.